search/house_to_street_table.cpp

```diff
@@ -32,15 +34,59 @@ class Fixed3BitsTable : public HouseToStreetTable
     return m_vector->Get(houseId, streetIndex);
   }
 
+  StreetIdType GetStreetIdType() const override { return StreetIdType::Index; }
+
+private:
+  unique_ptr<Vector> m_vector;
+};
+
+class EliasFanoMap : public HouseToStreetTable
+{
+public:
+  using Map = MapUint32ToValue<uint32_t>;
+
+  explicit EliasFanoMap(MwmValue & value) : m_reader(unique_ptr<ModelReader>())
+  {
+    auto const readBlockCallback = [&](NonOwningReaderSource & source, uint32_t blockSize,
+                                       vector<uint32_t> & values) {
+      values.resize(blockSize);
```

Comment:
At least assertion that blockSize is non-empty.

Reply:
done